_posts/en/posts/2016-08-05-release-0-13-0.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Bitcoin Core 0.13.0 Released!
+name: blog-release-0.13.0
+id: en-blog-release-0.13.0
+lang: en
+permalink: /en/2016/08/05/release-0.13.0/
+type: posts
+layout: post
+share: true
+version: 1
+---
+{% include _toc.html %}
+{% include _references.md %}
+
+We're pleased to announce the official release of Bitcoin Core 0.13.0.  During the six-month development cycle leading to this release, [dozens of contributors][release notes contributors] have made [hundreds of notable improvements][release notes changelog] to Bitcoin Core.  Among the many upgrades available in this release, the following may be especially interesting to miners, node operators, and wallet users:
+
+- **Preparation for segregated witness** to increase capacity, eliminate unwanted transaction malleability, and enable new ways to upgrade Bitcoin's Script language using soft forks.  The code in this release prepares for segwit only; it does not activate segwit on mainnet and will not behave differently when segwit is activated.
```

Comment:
"will not behave differently when segwit is activated" sounds confusing. What about "It does not support segwit on mainnet, so upgrading to a future version will be needed"?